cmd/arktos-network-controller/network-controller.go

```diff
@@ -90,4 +104,6 @@ func init() {
 	flag.StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
 	flag.IntVar(&workers, "concurrent-workers", defaultWorkers, "The number of workers that are allowed to process concurrently.")
 	flag.StringVar(&domainName, "cluster-domain", "cluster.local", "the cluster-internal domain name for Services.")
+	flag.StringVar(&kubeAPIServerIP, "kube-apiserver-ip", "", "the ip address kube-apiserver is listening at.")
```

Comment:
This is not a sustainable design to get kube api server ip from argument.

Reply:
agreed; issue #745 has been opened for the long-term fix.